debian/clearwater-auto-config-aws.init.d

```diff
@@ -72,6 +72,9 @@ do_auto_config()
   # Sprout will replace the cluster-settings file with something appropriate when it starts
   rm -f /etc/clearwater/cluster_settings
 
+  # Ensure that we start a clean cluster, so that cloned AIO nodes etc. boot cleanly
+  rm -rf /var/lib/clearwater-etcd/
```

Comment:
@eleanor-merry Given that the code now re-creates the directory, any reason to _not_ remove `/` ??